build.gradle

```diff
@@ -45,10 +45,6 @@ allprojects {
             indentWithSpaces()
             trimTrailingWhitespace()
         }
-        format 'styling', {
-            target '**/*.css'
-            prettier().config(['parser': 'postcss'])
```

Comment:
The parser is now just called `css`, in case you want to change it

Reply:
No joy simply changing it:

```
Execution failed for task ':spotlessStyling'.
> Unexpected response status code at /prettier/format [HTTP 501] -- (Error while formatting: TypeError: text.trimEnd is not a function)
```

Reply:
Ah, it's this error again. It seems like the newer 'prettier' requires a newer version of NodeJS, which introduces a `String.trimEnd` function.
But nevermind then